### PR TITLE
add remote host auth secret to zombie slots pruner cronjob

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: tasks/configure_namespace.yml [hunk 1]
```diff
@@ -206,11 +206,17 @@
                     volumeMounts:
                       - name: reactor-config-map
                         mountPath: /rcm
+                      - name: remote-host-auth
+                        mountPath: "{{ zombie_slots_pruner_remote_host_mount_path }}"
```
Comment (chmeliik) [inline tasks/configure_namespace.yml:210]: Could you document that the mountPath has to be the basename of all `auth` keys for all remote hosts in the configmap?

Side note: is it just me, or is the upstream role getting impossible to use without complete downstream knowledge?

File: README.md [hunk 2]
```diff
@@ -80,6 +80,12 @@ Role Variables
     # Config map for zombie slots pruner
     zombie_slots_pruner_rcm
 
+    # remote host auth secret for zombee slots pruner
+    zombie_slots_pruner_remote_host_secret
+
+    # remote host auth mount path for zombee slots pruner
+    zombie_slots_pruner_remote_host_mount_path
```
Comment (chmeliik) [inline README.md:87]: might deserve a rename, we're not mounting remote hosts, just their auth keys

File: README.md [hunk 3]
```diff
@@ -80,6 +80,12 @@ Role Variables
     # Config map for zombie slots pruner
     zombie_slots_pruner_rcm
 
+    # remote host auth secret for zombee slots pruner
+    zombie_slots_pruner_remote_host_secret
+
+    # remote host auth mount path for zombee slots pruner
```
[the remark this answers is not in the document]
Comment (rcerven) [reply README.md:86]: ok

File: tasks/configure_namespace.yml (hunk 1 above)
Comment (rcerven) [reply tasks/configure_namespace.yml:210]: updated

well the role depends on config map ....

File: README.md (hunk 2 above)
Comment (rcerven) [reply README.md]: added 'auth' there

**Review by hjmodi — APPROVED** [review]

**Review by ben-alkov — APPROVED** [review]

**Pull request merged**